README.md

```diff
@@ -1,107 +1,163 @@
-#ID.me WebVerify SDK (Android)
-The ID.me WebVerify SDK for Android is a class that allows you to verify a user's group affiliation status using ID.me's platform. A sample project has been provided to delineate the integration process.
+# ID.me WebVerify SDK (Android)
+The ID.me WebVerify SDK for Android is a library that allows you to verify a user's group affiliation status using ID.me's platform. A sample project has been provided to delineate the integration process.
 
-#Release Information 
-- **SDK Version:** 1.2.0 (August 11, 2016)
+## Specification
+### Supported Android Versions
+
+Android API 16 (Jellybean) and above is supported.
+
+## Release Information 
```

Comment:
What do you think about adding a link to the changelog here? https://github.com/IDme/ID.me-WebVerify-SDK-iOS/pull/34/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11